gensim/models/doc2vec.py

```diff
@@ -580,7 +580,7 @@ def __init__(self, documents=None, dm_mean=None,
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-                default is 0 (off), useful value is 1e-5.
+                default is 1e-3, useful value is 1e-5.
```

Comment:
I think the 'useful value' remark dates back to something in word2vec.c code, before it had a (modest) non-zero default. I suppose we could stop at just mentioning the default, or hint that more aggressive values are also worth trying with a comment like: "default is 1e-3; values to 1e-5 (or lower) may also be useful". (On large datasets, I've seen ok results with 1e-06 and 1e-07, which each offer additional training speedups.)